src/envs/behavior.py

```diff
@@ -1,32 +1,40 @@
 """Behavior (iGibson) environment."""
-# pylint: disable=import-error
+# pylint: disable=import-error,ungrouped-imports
```

Comment:
i'm confused, i thought in the previous round of revisions we agreed that we could get rid of disabling the `ungrouped-imports` check?

Reply:
indeed; idk how it snuck back in here (I must have forgotten to take it out!)